src/Games/NexusMods.Games.RedEngine/Cyberpunk2077Synchronizer.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Extensions.DependencyInjection;
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Loadouts.Synchronizers;
+using NexusMods.Abstractions.Settings;
+
+namespace NexusMods.Games.RedEngine;
+
+public class Cyberpunk2077Synchronizer : ALoadoutSynchronizer
+{
+    private readonly Cyberpunk2077Settings _settings;
+    protected internal Cyberpunk2077Synchronizer(IServiceProvider provider) : base(provider)
+    {
+        _settings = provider.GetRequiredService<ISettingsManager>().Get<Cyberpunk2077Settings>();
+    }
```

Comment:
```suggestion
    private Cyberpunk2077Settings _settings;

    protected internal Cyberpunk2077Synchronizer(IServiceProvider provider) : base(provider)
    {
        var settingsManager = provider.GetRequiredService<ISettingsManager>();

        _settings = settingsManager.Get<Cyberpunk2077Settings>();
        settingsManager.GetChanges<Cyberpunk2077Settings>().Subscribe(value => _settings = value);
    }
```

Otherwise, updates to the settings won't be seen until the next restart.